app/services/list_volunteer_opportunities.rb

```diff
@@ -0,0 +1,59 @@
+class ListVolunteerOpportunities
+
+  def self.with(listener, organisations, feature = Feature, http_party = HTTParty)
+    new(listener, organisations, feature, http_party).send(:run)
+  end
+
+  private
+
+  attr_reader :organisations, :listener, :feature, :http_party
+
+  def initialize(listener, organisations, feature, http_party)
+    @listener = listener
+    @organisations = organisations
+    @feature = feature
+    @http_party = http_party
+  end
+
+  def run
+    harrow_markers = listener.build_map_markers(organisations)
+    return [harrow_markers, []] unless feature.active? :doit_volunteer_opportunities
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)